spec/controllers/idv/inherited_proofing_cancellations_controller_spec.rb

```diff
@@ -145,17 +168,36 @@
       end
 
       it 'redirects to go back path' do
-        put :update, params: { step: step, cancel: 'true' }
+        action
 
         expect(response).to redirect_to go_back_path
       end
     end
+
+    it 'tracks the event in analytics' do
+      stub_analytics
+      request.env['HTTP_REFERER'] = 'https://example.com/'
+
+      expect(@analytics).to receive(:track_event).with(
+        'IdV: cancellation go back',
+        request_came_from: 'users/sessions#new',
+        step: step.to_s,
+        proofing_components: nil,
+        analytics_id: nil,
```

Comment:
I may be missing something; but it looks like `analytics_id` is being passed via `InheritedProofingCancellationsController#analytics_properties` when logged' is `analytics_id == nil` actually expected here (also in the other tests)?